include/cling/Interpreter/RuntimePrintValue.h

```diff
@@ -15,6 +15,7 @@
 #endif
 
 #include <string>
+#include <vector>
```

Comment:
@marsupial what ambiguity on Windows do you mean? This code is only used at cling's runtime, i.e. this will be only compiled by cling.